css/stylemenu.css

```diff
@@ -0,0 +1,69 @@
+.mobile-menu {
+  height: 0%;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background: #6070ff;
+  overflow-y: hidden;
+  overflow-x: hidden;
+  transition: 0.5s;
+}
```

Comment:
- Great Job for implementing the functionality 👍 however you need to adjust the background blur, font size, and position of the close button and menu list in your design to stick to the Figma design and have better UI 👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/231574732-8a73c979-0511-4ee5-b4a3-e8253b946099.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/231574722-f6e77559-2833-43a5-8e05-af958ad72159.png" width="200">|


😎️

🤝️